I 03-17 21:16:14 controller.py:59] DAG:
I 03-17 21:16:14 controller.py:59] [Task<name=compute-text-vectors>(run='# Initialize and dow...')
I 03-17 21:16:14 controller.py:59]   resources: {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}]
I 03-17 21:16:14 controller.py:195] Submitted managed job 51 (task: 0, name: 'compute-text-vectors'); SKYPILOT_TASK_ID: sky-managed-2025-03-17-21-16-14-679834_compute-text-vectors_51-0
I 03-17 21:16:14 controller.py:199] Started monitoring.
I 03-17 21:16:14 state.py:451] Launching the spot cluster...
I 03-17 21:16:15 storage.py:1058] Storage type StoreType.S3 already exists.
<sky-payload>"<rich_init>[bold cyan]Optimizing[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Optimizing[/]</rich_start>"</sky-payload>
I 03-17 21:16:15 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:16:15 optimizer.py:749] [1mEstimated cost: [0m$0.2 / hour
I 03-17 21:16:15 optimizer.py:749] 
I 03-17 21:16:15 optimizer.py:884] Considered resources (1 node):
I 03-17 21:16:15 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:16:15 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:16:15 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:16:15 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-south-1b   0.22       [32m   âœ”[0m     
I 03-17 21:16:15 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:16:15 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Launching[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:16:16 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-south-1[0m (ap-south-1b).
W 03-17 21:26:05 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-south-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 21:26:05 cloud_vm_ray_backend.py:2097] 
W 03-17 21:26:05 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:26:06 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:26:06 optimizer.py:749] [1mEstimated cost: [0m$0.2 / hour
I 03-17 21:26:06 optimizer.py:749] 
I 03-17 21:26:06 optimizer.py:884] Considered resources (1 node):
I 03-17 21:26:06 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:26:06 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:26:06 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:26:06 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-south-1a   0.25       [32m   âœ”[0m     
I 03-17 21:26:06 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:26:06 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:26:06 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-south-1[0m (ap-south-1a).
W 03-17 21:37:20 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-south-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 21:37:20 cloud_vm_ray_backend.py:2097] 
W 03-17 21:37:20 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:37:20 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:37:20 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:37:20 optimizer.py:749] 
I 03-17 21:37:20 optimizer.py:884] Considered resources (1 node):
I 03-17 21:37:20 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:37:20 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 21:37:20 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:37:20 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-2d   0.25       [32m   âœ”[0m     
I 03-17 21:37:20 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 21:37:20 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:37:20 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-2[0m (ap-northeast-2d).
[2mW 03-17 21:45:59 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 21:46:01 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-northeast-2d for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 21:46:01 cloud_vm_ray_backend.py:2097] 
W 03-17 21:46:01 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:46:01 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:46:01 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:46:01 optimizer.py:749] 
I 03-17 21:46:01 optimizer.py:884] Considered resources (1 node):
I 03-17 21:46:01 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:46:01 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 21:46:01 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 21:46:01 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-2a   0.27       [32m   âœ”[0m     
I 03-17 21:46:01 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 21:46:01 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:46:01 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-2[0m (ap-northeast-2a).
[2mW 03-17 21:57:13 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 21:57:15 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-northeast-2a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 21:57:15 cloud_vm_ray_backend.py:2097] 
W 03-17 21:57:15 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 21:57:15 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 21:57:15 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 21:57:15 optimizer.py:749] 
I 03-17 21:57:15 optimizer.py:884] Considered resources (1 node):
I 03-17 21:57:15 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:57:15 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 21:57:15 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 21:57:15 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1b    0.30       [32m   âœ”[0m     
I 03-17 21:57:15 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 21:57:15 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 21:57:15 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1b).
W 03-17 22:04:59 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:04:59 cloud_vm_ray_backend.py:2097] 
W 03-17 22:04:59 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:04:59 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:04:59 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:04:59 optimizer.py:749] 
I 03-17 22:04:59 optimizer.py:884] Considered resources (1 node):
I 03-17 22:04:59 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:04:59 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:04:59 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:04:59 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-2c    0.31       [32m   âœ”[0m     
I 03-17 22:04:59 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:04:59 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:04:59 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2c).
[2mW 03-17 22:16:21 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (us-east-2c). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing us-east-2a, us-east-2b.[0m
W 03-17 22:16:21 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-2c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:16:21 cloud_vm_ray_backend.py:2097] 
W 03-17 22:16:21 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:16:21 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:16:21 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:16:21 optimizer.py:749] 
I 03-17 22:16:21 optimizer.py:884] Considered resources (1 node):
I 03-17 22:16:21 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:16:21 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:16:21 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:16:21 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-2b    0.32       [32m   âœ”[0m     
I 03-17 22:16:21 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:16:21 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:16:21 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2b).
[2mW 03-17 22:18:38 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:18:38 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-2b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:18:38 cloud_vm_ray_backend.py:2097] 
W 03-17 22:18:38 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:18:39 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:18:39 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:18:39 optimizer.py:749] 
I 03-17 22:18:39 optimizer.py:884] Considered resources (1 node):
I 03-17 22:18:39 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:18:39 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 22:18:39 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:18:39 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ca-central-1a   0.33       [32m   âœ”[0m     
I 03-17 22:18:39 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 22:18:39 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:18:39 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ca-central-1[0m (ca-central-1a).
[2mW 03-17 22:18:50 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:18:51 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ca-central-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:18:51 cloud_vm_ray_backend.py:2097] 
W 03-17 22:18:51 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:18:51 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:18:51 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:18:51 optimizer.py:749] 
I 03-17 22:18:51 optimizer.py:884] Considered resources (1 node):
I 03-17 22:18:51 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:18:51 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:18:51 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:18:51 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1c    0.34       [32m   âœ”[0m     
I 03-17 22:18:51 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:18:51 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:18:51 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1c).
[2mE 03-17 22:18:52 config.py:46] SKYPILOT_ERROR_NO_NODES_LAUNCHED: No usable subnets matching availability zone ['us-east-1c'] found. Choose a different availability zone or try manually creating an instance in your specified region to populate the list of subnets and trying this again. If you have set `use_internal_ips`, check that this zone has a subnet that (1) has the substring 'private' in its name tag and (2) does not assign public IPs (`map_public_ip_on_launch` is False).[0m
W 03-17 22:18:53 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:18:53 cloud_vm_ray_backend.py:2097] 
W 03-17 22:18:53 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:18:53 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:18:53 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:18:53 optimizer.py:749] 
I 03-17 22:18:53 optimizer.py:884] Considered resources (1 node):
I 03-17 22:18:53 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:18:53 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 22:18:53 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:18:53 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ca-central-1b   0.34       [32m   âœ”[0m     
I 03-17 22:18:53 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 22:18:53 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:18:53 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ca-central-1[0m (ca-central-1b).
[2mW 03-17 22:19:05 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:19:05 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ca-central-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:19:05 cloud_vm_ray_backend.py:2097] 
W 03-17 22:19:05 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:19:05 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:19:05 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:19:05 optimizer.py:749] 
I 03-17 22:19:05 optimizer.py:884] Considered resources (1 node):
I 03-17 22:19:05 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:19:05 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:19:05 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:19:05 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-2a    0.34       [32m   âœ”[0m     
I 03-17 22:19:05 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:19:05 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:19:06 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-2[0m (eu-west-2a).
[2mW 03-17 22:19:29 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:19:30 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-2a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:19:30 cloud_vm_ray_backend.py:2097] 
W 03-17 22:19:30 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:19:30 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:19:30 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:19:30 optimizer.py:749] 
I 03-17 22:19:30 optimizer.py:884] Considered resources (1 node):
I 03-17 22:19:30 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:19:30 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:19:30 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:19:30 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-3c    0.35       [32m   âœ”[0m     
I 03-17 22:19:30 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:19:30 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:19:30 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-3[0m (eu-west-3c).
[2mW 03-17 22:19:37 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 22:19:38 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-3c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:19:38 cloud_vm_ray_backend.py:2097] 
W 03-17 22:19:38 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:19:38 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:19:38 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:19:38 optimizer.py:749] 
I 03-17 22:19:38 optimizer.py:884] Considered resources (1 node):
I 03-17 22:19:38 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:19:38 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:19:38 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:19:38 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-north-1a   0.35       [32m   âœ”[0m     
I 03-17 22:19:38 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:19:38 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:19:39 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-north-1[0m (eu-north-1a).
[2mW 03-17 22:19:50 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 22:19:51 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-north-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:19:51 cloud_vm_ray_backend.py:2097] 
W 03-17 22:19:51 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:19:51 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:19:51 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:19:51 optimizer.py:749] 
I 03-17 22:19:51 optimizer.py:884] Considered resources (1 node):
I 03-17 22:19:51 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:19:51 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:19:51 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:19:51 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-2b    0.36       [32m   âœ”[0m     
I 03-17 22:19:51 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:19:51 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:19:51 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-2[0m (eu-west-2b).
[2mW 03-17 22:20:06 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:20:07 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-2b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:20:07 cloud_vm_ray_backend.py:2097] 
W 03-17 22:20:07 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:20:08 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:20:08 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:20:08 optimizer.py:749] 
I 03-17 22:20:08 optimizer.py:884] Considered resources (1 node):
I 03-17 22:20:08 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:20:08 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 22:20:08 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:20:08 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-southeast-2c   0.36       [32m   âœ”[0m     
I 03-17 22:20:08 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 22:20:08 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:20:08 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-southeast-2[0m (ap-southeast-2c).
[2mW 03-17 22:20:25 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-southeast-2c). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-southeast-2a.[0m
W 03-17 22:20:26 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-southeast-2c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:20:26 cloud_vm_ray_backend.py:2097] 
W 03-17 22:20:26 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:20:27 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:20:27 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:20:27 optimizer.py:749] 
I 03-17 22:20:27 optimizer.py:884] Considered resources (1 node):
I 03-17 22:20:27 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:20:27 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:20:27 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:20:27 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1d    0.37       [32m   âœ”[0m     
I 03-17 22:20:27 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:20:27 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:20:27 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1d).
[2mW 03-17 22:20:39 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:20:40 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1d for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:20:40 cloud_vm_ray_backend.py:2097] 
W 03-17 22:20:40 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:20:40 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:20:40 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:20:40 optimizer.py:749] 
I 03-17 22:20:40 optimizer.py:884] Considered resources (1 node):
I 03-17 22:20:40 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:20:40 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:20:40 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:20:40 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-west-2b    0.37       [32m   âœ”[0m     
I 03-17 22:20:40 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:20:40 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:20:40 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-west-2[0m (us-west-2b).
[2mW 03-17 22:22:52 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (us-west-2b). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing us-west-2a, us-west-2c, us-west-2d.[0m
W 03-17 22:22:53 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-west-2b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:22:53 cloud_vm_ray_backend.py:2097] 
W 03-17 22:22:53 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:22:54 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:22:54 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:22:54 optimizer.py:749] 
I 03-17 22:22:54 optimizer.py:884] Considered resources (1 node):
I 03-17 22:22:54 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:22:54 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:22:54 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:22:54 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-north-1b   0.37       [32m   âœ”[0m     
I 03-17 22:22:54 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:22:54 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:22:54 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-north-1[0m (eu-north-1b).
[2mW 03-17 22:23:05 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 22:23:06 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-north-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:23:06 cloud_vm_ray_backend.py:2097] 
W 03-17 22:23:06 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:23:06 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:23:06 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:23:06 optimizer.py:749] 
I 03-17 22:23:06 optimizer.py:884] Considered resources (1 node):
I 03-17 22:23:06 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:23:06 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 22:23:06 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:23:06 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-southeast-2a   0.38       [32m   âœ”[0m     
I 03-17 22:23:06 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 22:23:06 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:23:06 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-southeast-2[0m (ap-southeast-2a).
[2mW 03-17 22:23:21 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-southeast-2a). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-southeast-2c.[0m
W 03-17 22:23:23 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-southeast-2a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:23:23 cloud_vm_ray_backend.py:2097] 
W 03-17 22:23:23 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:23:23 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:23:23 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:23:23 optimizer.py:749] 
I 03-17 22:23:23 optimizer.py:884] Considered resources (1 node):
I 03-17 22:23:23 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:23:23 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:23:23 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:23:23 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-3b    0.38       [32m   âœ”[0m     
I 03-17 22:23:23 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:23:23 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:23:23 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-3[0m (eu-west-3b).
[2mW 03-17 22:23:30 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (eu-west-3b). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing eu-west-3c.[0m
W 03-17 22:23:31 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-3b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:23:31 cloud_vm_ray_backend.py:2097] 
W 03-17 22:23:31 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:23:31 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:23:31 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:23:31 optimizer.py:749] 
I 03-17 22:23:31 optimizer.py:884] Considered resources (1 node):
I 03-17 22:23:31 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:23:31 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:23:31 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:23:31 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1a    0.39       [32m   âœ”[0m     
I 03-17 22:23:31 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:23:31 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:23:31 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1a).
[2mW 03-17 22:23:37 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:23:38 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:23:38 cloud_vm_ray_backend.py:2097] 
W 03-17 22:23:38 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:23:38 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:23:38 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:23:38 optimizer.py:749] 
I 03-17 22:23:38 optimizer.py:884] Considered resources (1 node):
I 03-17 22:23:38 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:23:38 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:23:38 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:23:38 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-2a    0.40       [32m   âœ”[0m     
I 03-17 22:23:38 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:23:38 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:23:38 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2a).
[2mW 03-17 22:24:15 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (us-east-2a). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing us-east-2b, us-east-2c.[0m
W 03-17 22:24:16 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-2a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:24:16 cloud_vm_ray_backend.py:2097] 
W 03-17 22:24:16 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:24:16 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:24:16 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:24:16 optimizer.py:749] 
I 03-17 22:24:16 optimizer.py:884] Considered resources (1 node):
I 03-17 22:24:16 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:24:16 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 22:24:16 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:24:16 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-central-1b   0.40       [32m   âœ”[0m     
I 03-17 22:24:16 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 22:24:16 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:24:16 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-central-1[0m (eu-central-1b).
[2mW 03-17 22:25:12 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:25:13 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-central-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:25:13 cloud_vm_ray_backend.py:2097] 
W 03-17 22:25:13 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:25:13 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:25:13 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:25:13 optimizer.py:749] 
I 03-17 22:25:13 optimizer.py:884] Considered resources (1 node):
I 03-17 22:25:13 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:25:13 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:25:13 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:25:13 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-west-2d    0.41       [32m   âœ”[0m     
I 03-17 22:25:13 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:25:13 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:25:13 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-west-2[0m (us-west-2d).
W 03-17 22:35:04 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-west-2d for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:35:04 cloud_vm_ray_backend.py:2097] 
W 03-17 22:35:04 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:35:04 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:35:04 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:35:04 optimizer.py:749] 
I 03-17 22:35:04 optimizer.py:884] Considered resources (1 node):
I 03-17 22:35:04 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:35:04 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 22:35:04 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:35:04 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-central-1a   0.41       [32m   âœ”[0m     
I 03-17 22:35:04 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 22:35:04 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:35:04 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-central-1[0m (eu-central-1a).
[2mW 03-17 22:35:18 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:35:20 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-central-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:35:20 cloud_vm_ray_backend.py:2097] 
W 03-17 22:35:20 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:35:20 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:35:20 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:35:20 optimizer.py:749] 
I 03-17 22:35:20 optimizer.py:884] Considered resources (1 node):
I 03-17 22:35:20 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:35:20 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:35:20 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:35:20 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-west-2c    0.41       [32m   âœ”[0m     
I 03-17 22:35:20 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:35:20 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:35:20 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-west-2[0m (us-west-2c).
W 03-17 22:45:04 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-west-2c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:45:04 cloud_vm_ray_backend.py:2097] 
W 03-17 22:45:04 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:45:04 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:45:04 optimizer.py:749] [1mEstimated cost: [0m$0.5 / hour
I 03-17 22:45:04 optimizer.py:749] 
I 03-17 22:45:04 optimizer.py:884] Considered resources (1 node):
I 03-17 22:45:04 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:45:04 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 22:45:04 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:45:04 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-1a   0.46       [32m   âœ”[0m     
I 03-17 22:45:04 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 22:45:04 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:45:04 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-1[0m (ap-northeast-1a).
[2mW 03-17 22:45:19 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-northeast-1a). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-northeast-1c.[0m
W 03-17 22:45:19 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-northeast-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:45:19 cloud_vm_ray_backend.py:2097] 
W 03-17 22:45:19 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:45:20 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:45:20 optimizer.py:749] [1mEstimated cost: [0m$0.5 / hour
I 03-17 22:45:20 optimizer.py:749] 
I 03-17 22:45:20 optimizer.py:884] Considered resources (1 node):
I 03-17 22:45:20 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:45:20 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 22:45:20 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:45:20 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-1c   0.48       [32m   âœ”[0m     
I 03-17 22:45:20 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 22:45:20 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:45:20 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-1[0m (ap-northeast-1c).
[2mW 03-17 22:45:42 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 22:45:43 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-northeast-1c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:45:43 cloud_vm_ray_backend.py:2097] 
W 03-17 22:45:43 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:45:43 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:45:43 optimizer.py:749] [1mEstimated cost: [0m$0.8 / hour
I 03-17 22:45:43 optimizer.py:749] 
I 03-17 22:45:43 optimizer.py:884] Considered resources (1 node):
I 03-17 22:45:43 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:45:43 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:45:43 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:45:43 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           us-east-1     0.80       [32m   âœ”[0m     
I 03-17 22:45:43 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:45:43 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in us-east-1 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:45:43 cloud_vm_ray_backend.py:2097] 
W 03-17 22:45:43 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:45:43 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:45:43 optimizer.py:749] [1mEstimated cost: [0m$0.8 / hour
I 03-17 22:45:43 optimizer.py:749] 
I 03-17 22:45:43 optimizer.py:884] Considered resources (1 node):
I 03-17 22:45:43 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:45:43 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:45:43 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:45:43 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           us-east-2     0.80       [32m   âœ”[0m     
I 03-17 22:45:43 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:45:43 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in us-east-2 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:45:43 cloud_vm_ray_backend.py:2097] 
W 03-17 22:45:43 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:45:43 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:45:43 optimizer.py:749] [1mEstimated cost: [0m$0.8 / hour
I 03-17 22:45:43 optimizer.py:749] 
I 03-17 22:45:43 optimizer.py:884] Considered resources (1 node):
I 03-17 22:45:43 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:45:43 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:45:43 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:45:43 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           us-west-2     0.80       [32m   âœ”[0m     
I 03-17 22:45:43 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:45:43 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-west-2[0m (us-west-2a).
[2mW 03-17 22:46:51 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (us-west-2a). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing us-west-2b, us-west-2c, us-west-2d.[0m
W 03-17 22:46:52 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in us-west-2 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:46:52 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:46:52 optimizer.py:749] [1mEstimated cost: [0m$0.9 / hour
I 03-17 22:46:52 optimizer.py:749] 
I 03-17 22:46:52 optimizer.py:884] Considered resources (1 node):
I 03-17 22:46:52 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:46:52 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:46:52 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:46:52 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           eu-north-1    0.85       [32m   âœ”[0m     
I 03-17 22:46:52 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:46:52 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in eu-north-1 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:46:52 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:46:52 optimizer.py:749] [1mEstimated cost: [0m$0.9 / hour
I 03-17 22:46:52 optimizer.py:749] 
I 03-17 22:46:52 optimizer.py:884] Considered resources (1 node):
I 03-17 22:46:52 optimizer.py:952] -----------------------------------------------------------------------------------------
I 03-17 22:46:52 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE    COST ($)   CHOSEN   
I 03-17 22:46:52 optimizer.py:952] -----------------------------------------------------------------------------------------
I 03-17 22:46:52 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ca-central-1   0.89       [32m   âœ”[0m     
I 03-17 22:46:52 optimizer.py:952] -----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:46:52 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in ca-central-1 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:46:52 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:46:52 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 22:46:52 optimizer.py:749] 
I 03-17 22:46:52 optimizer.py:884] Considered resources (1 node):
I 03-17 22:46:52 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:46:52 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:46:52 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:46:52 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ap-south-1    0.97       [32m   âœ”[0m     
I 03-17 22:46:52 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:46:52 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in ap-south-1 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:46:52 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:46:52 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 22:46:52 optimizer.py:749] 
I 03-17 22:46:52 optimizer.py:884] Considered resources (1 node):
I 03-17 22:46:52 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 22:46:52 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE      COST ($)   CHOSEN   
I 03-17 22:46:52 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 22:46:52 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ap-northeast-2   0.99       [32m   âœ”[0m     
I 03-17 22:46:52 optimizer.py:952] -------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:46:52 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in ap-northeast-2 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:46:52 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:46:52 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 22:46:52 optimizer.py:749] 
I 03-17 22:46:52 optimizer.py:884] Considered resources (1 node):
I 03-17 22:46:52 optimizer.py:952] -----------------------------------------------------------------------------------------
I 03-17 22:46:52 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE    COST ($)   CHOSEN   
I 03-17 22:46:52 optimizer.py:952] -----------------------------------------------------------------------------------------
I 03-17 22:46:52 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           eu-central-1   1.01       [32m   âœ”[0m     
I 03-17 22:46:52 optimizer.py:952] -----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:46:52 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in eu-central-1 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:46:52 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:46:52 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 22:46:52 optimizer.py:749] 
I 03-17 22:46:52 optimizer.py:884] Considered resources (1 node):
I 03-17 22:46:52 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:46:52 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:46:52 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:46:52 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           eu-west-2     1.02       [32m   âœ”[0m     
I 03-17 22:46:52 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:46:52 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in eu-west-2 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:46:52 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:46:52 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 22:46:52 optimizer.py:749] 
I 03-17 22:46:52 optimizer.py:884] Considered resources (1 node):
I 03-17 22:46:52 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:46:52 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:46:52 optimizer.py:952] ----------------------------------------------------------------------------------------
I 03-17 22:46:52 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           eu-west-3     1.02       [32m   âœ”[0m     
I 03-17 22:46:52 optimizer.py:952] ----------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:46:52 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in eu-west-3 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:46:52 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:46:52 optimizer.py:749] [1mEstimated cost: [0m$1.0 / hour
I 03-17 22:46:52 optimizer.py:749] 
I 03-17 22:46:52 optimizer.py:884] Considered resources (1 node):
I 03-17 22:46:52 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 22:46:52 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE      COST ($)   CHOSEN   
I 03-17 22:46:52 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 22:46:52 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ap-southeast-2   1.05       [32m   âœ”[0m     
I 03-17 22:46:52 optimizer.py:952] -------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:46:52 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in ap-southeast-2 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:46:52 optimizer.py:749] [1mEstimated cost: [0m$1.2 / hour
I 03-17 22:46:52 optimizer.py:749] 
I 03-17 22:46:52 optimizer.py:884] Considered resources (1 node):
I 03-17 22:46:52 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 22:46:52 optimizer.py:952]  CLOUD   INSTANCE    vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE      COST ($)   CHOSEN   
I 03-17 22:46:52 optimizer.py:952] -------------------------------------------------------------------------------------------
I 03-17 22:46:52 optimizer.py:952]  AWS     g6.xlarge   4       16        L4:1           ap-northeast-1   1.17       [32m   âœ”[0m     
I 03-17 22:46:52 optimizer.py:952] -------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
W 03-17 22:46:52 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in all zones in ap-northeast-1 for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] 
W 03-17 22:46:52 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log\u001b[0m</rich_update>"</sky-payload>
E 03-17 22:46:52 cloud_vm_ray_backend.py:2904] [0m[31mâ¨¯[0m Failed to provision resources. [2mView logs: sky api logs -l sky-2025-03-17-21-16-15-275490/provision.log[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>

[?25hI 03-17 22:46:53 recovery_strategy.py:326] Failed to launch a cluster with error: sky.exceptions.ResourcesUnavailableError: [31mFailed to provision all possible launchable resources.[0m Relax the task's resource requirements: 1x AWS([Spot], cpus=4, {'L4': 1})
I 03-17 22:46:53 recovery_strategy.py:326] To keep retrying until the cluster is up, use the `--retry-until-up` flag.
I 03-17 22:46:53 recovery_strategy.py:326] Reasons for provision failures (for details, please check the log above):
I 03-17 22:46:53 recovery_strategy.py:326] Resource         Reason                                                      
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ap-south-1b for              
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ap-south-1a for              
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ap-northeast-2d for          
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ap-northeast-2a for          
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-east-1b for {AWS([Spot],  
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-east-2c for {AWS([Spot],  
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-east-2b for {AWS([Spot],  
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ca-central-1a for            
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-east-1c for {AWS([Spot],  
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ca-central-1b for            
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in eu-west-2a for {AWS([Spot],  
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in eu-west-3c for {AWS([Spot],  
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in eu-north-1a for              
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in eu-west-2b for {AWS([Spot],  
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ap-southeast-2c for          
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-east-1d for {AWS([Spot],  
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-west-2b for {AWS([Spot],  
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in eu-north-1b for              
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ap-southeast-2a for          
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in eu-west-3b for {AWS([Spot],  
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-east-1a for {AWS([Spot],  
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-east-2a for {AWS([Spot],  
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in eu-central-1b for            
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-west-2d for {AWS([Spot],  
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in eu-central-1a for            
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in us-west-2c for {AWS([Spot],  
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.                
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ap-northeast-1a for          
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge[S  Failed to acquire resources in ap-northeast-1c for          
I 03-17 22:46:53 recovery_strategy.py:326] pot], {'L4':     {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] 1})                                                                          
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in us-east-1 for   
I 03-17 22:46:53 recovery_strategy.py:326] {'L4': 1})       {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in us-east-2 for   
I 03-17 22:46:53 recovery_strategy.py:326] {'L4': 1})       {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in us-west-2 for   
I 03-17 22:46:53 recovery_strategy.py:326] {'L4': 1})       {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in eu-north-1 for  
I 03-17 22:46:53 recovery_strategy.py:326] {'L4': 1})       {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in ca-central-1    
I 03-17 22:46:53 recovery_strategy.py:326] {'L4': 1})       for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4':     
I 03-17 22:46:53 recovery_strategy.py:326]                  1})}.                                                       
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in ap-south-1 for  
I 03-17 22:46:53 recovery_strategy.py:326] {'L4': 1})       {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in ap-northeast-2  
I 03-17 22:46:53 recovery_strategy.py:326] {'L4': 1})       for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4':     
I 03-17 22:46:53 recovery_strategy.py:326]                  1})}.                                                       
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in eu-central-1    
I 03-17 22:46:53 recovery_strategy.py:326] {'L4': 1})       for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4':     
I 03-17 22:46:53 recovery_strategy.py:326]                  1})}.                                                       
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in eu-west-2 for   
I 03-17 22:46:53 recovery_strategy.py:326] {'L4': 1})       {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in eu-west-3 for   
I 03-17 22:46:53 recovery_strategy.py:326] {'L4': 1})       {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}.   
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in ap-southeast-2  
I 03-17 22:46:53 recovery_strategy.py:326] {'L4': 1})       for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4':     
I 03-17 22:46:53 recovery_strategy.py:326]                  1})}.                                                       
I 03-17 22:46:53 recovery_strategy.py:326] AWS(g6.xlarge,   Failed to acquire resources in all zones in ap-northeast-1  
I 03-17 22:46:53 recovery_strategy.py:326] {'L4': 1})       for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4':     
I 03-17 22:46:53 recovery_strategy.py:326]                  1})}.                                                       )
I 03-17 22:46:53 recovery_strategy.py:364] Retrying to launch the cluster in 83.4 seconds.
I 03-17 22:48:16 storage.py:699] Verifying bucket for storage skypilot-filemounts-root-726f1974-czxty1fy
<sky-payload>"<rich_init>[bold cyan]Optimizing[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Optimizing[/]</rich_start>"</sky-payload>
I 03-17 22:48:17 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:48:17 optimizer.py:749] [1mEstimated cost: [0m$0.2 / hour
I 03-17 22:48:17 optimizer.py:749] 
I 03-17 22:48:17 optimizer.py:884] Considered resources (1 node):
I 03-17 22:48:17 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:48:17 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:48:17 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:48:17 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-south-1b   0.22       [32m   âœ”[0m     
I 03-17 22:48:17 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:48:17 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Launching[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:48:17 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-south-1[0m (ap-south-1b).
[2mW 03-17 22:48:30 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:48:31 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-south-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:48:31 cloud_vm_ray_backend.py:2097] 
W 03-17 22:48:31 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:48:31 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:48:31 optimizer.py:749] [1mEstimated cost: [0m$0.2 / hour
I 03-17 22:48:31 optimizer.py:749] 
I 03-17 22:48:31 optimizer.py:884] Considered resources (1 node):
I 03-17 22:48:31 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:48:31 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:48:31 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:48:31 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-south-1a   0.25       [32m   âœ”[0m     
I 03-17 22:48:31 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:48:31 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:48:31 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-south-1[0m (ap-south-1a).
[2mW 03-17 22:48:43 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:48:44 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-south-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:48:44 cloud_vm_ray_backend.py:2097] 
W 03-17 22:48:44 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:48:45 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:48:45 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:48:45 optimizer.py:749] 
I 03-17 22:48:45 optimizer.py:884] Considered resources (1 node):
I 03-17 22:48:45 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:48:45 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 22:48:45 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:48:45 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-2d   0.25       [32m   âœ”[0m     
I 03-17 22:48:45 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 22:48:45 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:48:45 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-2[0m (ap-northeast-2d).
[2mW 03-17 22:48:58 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 22:48:59 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-northeast-2d for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:48:59 cloud_vm_ray_backend.py:2097] 
W 03-17 22:48:59 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:49:00 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:49:00 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:49:00 optimizer.py:749] 
I 03-17 22:49:00 optimizer.py:884] Considered resources (1 node):
I 03-17 22:49:00 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:49:00 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 22:49:00 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:49:00 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-northeast-2a   0.27       [32m   âœ”[0m     
I 03-17 22:49:00 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 22:49:00 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:49:00 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-northeast-2[0m (ap-northeast-2a).
[2mW 03-17 22:49:11 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (ap-northeast-2a). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing ap-northeast-2d.[0m
W 03-17 22:49:13 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ap-northeast-2a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:49:13 cloud_vm_ray_backend.py:2097] 
W 03-17 22:49:13 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:49:13 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:49:13 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:49:13 optimizer.py:749] 
I 03-17 22:49:13 optimizer.py:884] Considered resources (1 node):
I 03-17 22:49:13 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:49:13 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:49:13 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:49:13 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1b    0.30       [32m   âœ”[0m     
I 03-17 22:49:13 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:49:13 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:49:13 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1b).
[2mW 03-17 22:49:26 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:49:27 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:49:27 cloud_vm_ray_backend.py:2097] 
W 03-17 22:49:27 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:49:27 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:49:27 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:49:27 optimizer.py:749] 
I 03-17 22:49:27 optimizer.py:884] Considered resources (1 node):
I 03-17 22:49:27 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:49:27 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:49:27 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:49:27 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-2c    0.31       [32m   âœ”[0m     
I 03-17 22:49:27 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:49:27 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:49:27 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2c).
[2mW 03-17 22:49:34 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): We currently do not have sufficient g6.xlarge capacity in the Availability Zone you requested (us-east-2c). Our system will be working on provisioning additional capacity. You can currently get g6.xlarge capacity by not specifying an Availability Zone in your request or choosing us-east-2a, us-east-2b.[0m
W 03-17 22:49:35 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-2c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:49:35 cloud_vm_ray_backend.py:2097] 
W 03-17 22:49:35 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:49:35 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:49:35 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:49:35 optimizer.py:749] 
I 03-17 22:49:35 optimizer.py:884] Considered resources (1 node):
I 03-17 22:49:35 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:49:35 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:49:35 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:49:35 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-2b    0.32       [32m   âœ”[0m     
I 03-17 22:49:35 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:49:35 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:49:35 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-2[0m (us-east-2b).
[2mW 03-17 22:49:42 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:49:43 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-2b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:49:43 cloud_vm_ray_backend.py:2097] 
W 03-17 22:49:43 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:49:43 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:49:43 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:49:43 optimizer.py:749] 
I 03-17 22:49:43 optimizer.py:884] Considered resources (1 node):
I 03-17 22:49:43 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:49:43 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 22:49:43 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:49:43 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ca-central-1a   0.33       [32m   âœ”[0m     
I 03-17 22:49:43 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 22:49:43 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:49:43 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ca-central-1[0m (ca-central-1a).
[2mW 03-17 22:49:48 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:49:48 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ca-central-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:49:48 cloud_vm_ray_backend.py:2097] 
W 03-17 22:49:48 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:49:49 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:49:49 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:49:49 optimizer.py:749] 
I 03-17 22:49:49 optimizer.py:884] Considered resources (1 node):
I 03-17 22:49:49 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:49:49 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:49:49 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:49:49 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           us-east-1c    0.34       [32m   âœ”[0m     
I 03-17 22:49:49 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:49:49 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:49:49 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS us-east-1[0m (us-east-1c).
[2mE 03-17 22:49:50 config.py:46] SKYPILOT_ERROR_NO_NODES_LAUNCHED: No usable subnets matching availability zone ['us-east-1c'] found. Choose a different availability zone or try manually creating an instance in your specified region to populate the list of subnets and trying this again. If you have set `use_internal_ips`, check that this zone has a subnet that (1) has the substring 'private' in its name tag and (2) does not assign public IPs (`map_public_ip_on_launch` is False).[0m
W 03-17 22:49:51 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in us-east-1c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:49:51 cloud_vm_ray_backend.py:2097] 
W 03-17 22:49:51 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:49:51 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:49:51 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:49:51 optimizer.py:749] 
I 03-17 22:49:51 optimizer.py:884] Considered resources (1 node):
I 03-17 22:49:51 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:49:51 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE     COST ($)   CHOSEN   
I 03-17 22:49:51 optimizer.py:952] ------------------------------------------------------------------------------------------------
I 03-17 22:49:51 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ca-central-1b   0.34       [32m   âœ”[0m     
I 03-17 22:49:51 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1       0.80                
I 03-17 22:49:51 optimizer.py:952] ------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:49:51 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ca-central-1[0m (ca-central-1b).
[2mW 03-17 22:50:02 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:50:02 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in ca-central-1b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:50:02 cloud_vm_ray_backend.py:2097] 
W 03-17 22:50:02 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:50:03 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:50:03 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:50:03 optimizer.py:749] 
I 03-17 22:50:03 optimizer.py:884] Considered resources (1 node):
I 03-17 22:50:03 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:50:03 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:50:03 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:50:03 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-2a    0.34       [32m   âœ”[0m     
I 03-17 22:50:03 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:50:03 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:50:03 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-2[0m (eu-west-2a).
[2mW 03-17 22:50:10 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:50:11 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-2a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:50:11 cloud_vm_ray_backend.py:2097] 
W 03-17 22:50:11 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:50:12 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:50:12 optimizer.py:749] [1mEstimated cost: [0m$0.3 / hour
I 03-17 22:50:12 optimizer.py:749] 
I 03-17 22:50:12 optimizer.py:884] Considered resources (1 node):
I 03-17 22:50:12 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:50:12 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:50:12 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:50:12 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-3c    0.35       [32m   âœ”[0m     
I 03-17 22:50:12 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:50:12 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:50:12 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-3[0m (eu-west-3c).
[2mW 03-17 22:50:18 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 22:50:20 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-3c for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:50:20 cloud_vm_ray_backend.py:2097] 
W 03-17 22:50:20 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:50:20 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:50:20 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:50:20 optimizer.py:749] 
I 03-17 22:50:20 optimizer.py:884] Considered resources (1 node):
I 03-17 22:50:20 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:50:20 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:50:20 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:50:20 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-north-1a   0.35       [32m   âœ”[0m     
I 03-17 22:50:20 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:50:20 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:50:20 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-north-1[0m (eu-north-1a).
[2mW 03-17 22:50:38 instance.py:243] create_instances: Attempt failed with An error occurred (InsufficientInstanceCapacity) when calling the RunInstances operation (reached max retries: 0): There is no Spot capacity available that matches your request.[0m
W 03-17 22:50:39 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-north-1a for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:50:39 cloud_vm_ray_backend.py:2097] 
W 03-17 22:50:39 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:50:39 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:50:39 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:50:39 optimizer.py:749] 
I 03-17 22:50:39 optimizer.py:884] Considered resources (1 node):
I 03-17 22:50:39 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:50:39 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE   COST ($)   CHOSEN   
I 03-17 22:50:39 optimizer.py:952] ----------------------------------------------------------------------------------------------
I 03-17 22:50:39 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           eu-west-2b    0.36       [32m   âœ”[0m     
I 03-17 22:50:39 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1     0.80                
I 03-17 22:50:39 optimizer.py:952] ----------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:50:40 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS eu-west-2[0m (eu-west-2b).
[2mW 03-17 22:50:47 instance.py:243] create_instances: Attempt failed with An error occurred (MaxSpotInstanceCountExceeded) when calling the RunInstances operation: Max spot instance count exceeded[0m
W 03-17 22:50:48 cloud_vm_ray_backend.py:2061] sky.exceptions.ResourcesUnavailableError: Failed to acquire resources in eu-west-2b for {AWS([Spot], cpus=4, {'L4': 1}), AWS(cpus=4, {'L4': 1})}. 
W 03-17 22:50:48 cloud_vm_ray_backend.py:2097] 
W 03-17 22:50:48 cloud_vm_ray_backend.py:2097] [0m[33mâ†º[0m Trying other potential resources.
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Optimizing[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:50:48 optimizer.py:736] [1mTarget:[0m minimizing cost
I 03-17 22:50:48 optimizer.py:749] [1mEstimated cost: [0m$0.4 / hour
I 03-17 22:50:48 optimizer.py:749] 
I 03-17 22:50:48 optimizer.py:884] Considered resources (1 node):
I 03-17 22:50:48 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:50:48 optimizer.py:952]  CLOUD   INSTANCE          vCPUs   Mem(GB)   ACCELERATORS   REGION/ZONE       COST ($)   CHOSEN   
I 03-17 22:50:48 optimizer.py:952] --------------------------------------------------------------------------------------------------
I 03-17 22:50:48 optimizer.py:952]  AWS     g6.xlarge[Spot]   4       16        L4:1           ap-southeast-2c   0.36       [32m   âœ”[0m     
I 03-17 22:50:48 optimizer.py:952]  AWS     g6.xlarge         4       16        L4:1           us-east-1         0.80                
I 03-17 22:50:48 optimizer.py:952] --------------------------------------------------------------------------------------------------
<sky-payload>"<rich_update>[bold cyan]Looking for resources[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:50:48 cloud_vm_ray_backend.py:1520] [0mâš™ï¸Ž Launching on AWS ap-southeast-2[0m (ap-southeast-2c).
<sky-payload>"<rich_update>[bold cyan]Launching - Checking instance status[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching - Waiting for SSH access[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Launching[/]</rich_start>"</sky-payload>
I 03-17 22:51:21 provisioner.py:451] [2mâ””â”€â”€ [0m[2mInstance is up.[0m
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (1/3 - initializing)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (2/3 - dependencies)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Preparing SkyPilot runtime (3/3 - runtime)[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Launching - Checking instance status[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:51:50 provisioner.py:632] [0m[32mâœ“ Cluster launched: compute-text-vectors-51.[0m  [2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/provision.log[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
I 03-17 22:51:50 execution.py:332] [0mâš™ï¸Ž Syncing files.
<sky-payload>"<rich_init>[bold cyan]Syncing files[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Syncing files[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Syncing files[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Syncing file mounts[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/file_mounts.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:51:50 backend_utils.py:1315]   [2mSyncing (to 1 node): s3://skypilot-filemounts-root-726f1974-czxty1fy/job-czxty1fy/workdir -> ~/sky_workdir[0m
I 03-17 22:51:54 cloud_vm_ray_backend.py:4801] [0m[32mâœ“ Synced file_mounts.[0m  [2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/file_mounts.log[0m
<sky-payload>"<rich_update>[bold cyan]Mounting 1 storage[/]  \u001b[2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/storage_mounts.log\u001b[0m</rich_update>"</sky-payload>
I 03-17 22:51:54 backend_utils.py:1315]   [2mMounting (to 1 node): s3://sky-text-embeddings-multi-cluster-final-2 -> /output[0m
I 03-17 22:51:58 cloud_vm_ray_backend.py:4892] [0m[32mâœ“ Storage mounted.[0m  [2mView logs: sky api logs -l sky-2025-03-17-22-48-17-127262/storage_mounts.log[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Running setup[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Running setup[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Running setup[/]</rich_start>"</sky-payload>
<sky-payload>"<rich_stop></rich_stop>"</sky-payload>
I 03-17 22:52:02 cloud_vm_ray_backend.py:3357] [0m[32mâœ“ Setup detached.[0m
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>
<sky-payload>"<rich_init>[bold cyan]Submitting job[/]</rich_init>"</sky-payload>
<sky-payload>"<rich_update>[bold cyan]Submitting job[/]</rich_update>"</sky-payload>
<sky-payload>"<rich_start>[bold cyan]Submitting job[/]</rich_start>"</sky-payload>
I 03-17 22:52:03 cloud_vm_ray_backend.py:3546] Multiple resources are specified for the task, using: AWS([Spot], cpus=4, {'L4': 1})
I 03-17 22:52:05 cloud_vm_ray_backend.py:3460] [0mâš™ï¸Ž Job submitted, ID: 1
<sky-payload>"<rich_stop></rich_stop>"</sky-payload>
<sky-payload>"<rich_exit></rich_exit>"</sky-payload>

[?25hI 03-17 22:52:05 recovery_strategy.py:289] Managed job cluster launched.
I 03-17 22:52:09 utils.py:131] === Checking the job status... ===
I 03-17 22:52:10 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 22:52:10 utils.py:140] ==================================
I 03-17 22:52:11 state.py:471] Job started.
I 03-17 22:52:31 utils.py:131] === Checking the job status... ===
I 03-17 22:52:32 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 22:52:32 utils.py:140] ==================================
I 03-17 22:52:52 utils.py:131] === Checking the job status... ===
I 03-17 22:52:54 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 22:52:54 utils.py:140] ==================================
I 03-17 22:53:14 utils.py:131] === Checking the job status... ===
I 03-17 22:53:15 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 22:53:15 utils.py:140] ==================================
I 03-17 22:53:35 utils.py:131] === Checking the job status... ===
I 03-17 22:53:36 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 22:53:36 utils.py:140] ==================================
I 03-17 22:53:56 utils.py:131] === Checking the job status... ===
I 03-17 22:53:57 utils.py:137] Job status: JobStatus.SETTING_UP
I 03-17 22:53:57 utils.py:140] ==================================
I 03-17 22:54:17 utils.py:131] === Checking the job status... ===
I 03-17 22:54:19 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:54:19 utils.py:140] ==================================
I 03-17 22:54:39 utils.py:131] === Checking the job status... ===
I 03-17 22:54:40 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:54:40 utils.py:140] ==================================
I 03-17 22:55:00 utils.py:131] === Checking the job status... ===
I 03-17 22:55:02 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:55:02 utils.py:140] ==================================
I 03-17 22:55:22 utils.py:131] === Checking the job status... ===
I 03-17 22:55:23 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:55:23 utils.py:140] ==================================
I 03-17 22:55:43 utils.py:131] === Checking the job status... ===
I 03-17 22:55:44 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:55:44 utils.py:140] ==================================
I 03-17 22:56:04 utils.py:131] === Checking the job status... ===
I 03-17 22:56:05 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:56:05 utils.py:140] ==================================
I 03-17 22:56:25 utils.py:131] === Checking the job status... ===
I 03-17 22:56:26 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:56:26 utils.py:140] ==================================
I 03-17 22:56:47 utils.py:131] === Checking the job status... ===
I 03-17 22:56:48 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:56:48 utils.py:140] ==================================
I 03-17 22:57:08 utils.py:131] === Checking the job status... ===
I 03-17 22:57:09 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:57:09 utils.py:140] ==================================
I 03-17 22:57:29 utils.py:131] === Checking the job status... ===
I 03-17 22:57:30 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:57:30 utils.py:140] ==================================
I 03-17 22:57:50 utils.py:131] === Checking the job status... ===
I 03-17 22:57:54 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:57:54 utils.py:140] ==================================
I 03-17 22:58:14 utils.py:131] === Checking the job status... ===
I 03-17 22:58:15 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:58:15 utils.py:140] ==================================
I 03-17 22:58:35 utils.py:131] === Checking the job status... ===
I 03-17 22:58:39 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:58:39 utils.py:140] ==================================
I 03-17 22:58:59 utils.py:131] === Checking the job status... ===
I 03-17 22:59:00 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:59:00 utils.py:140] ==================================
I 03-17 22:59:20 utils.py:131] === Checking the job status... ===
I 03-17 22:59:21 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:59:21 utils.py:140] ==================================
I 03-17 22:59:41 utils.py:131] === Checking the job status... ===
I 03-17 22:59:42 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 22:59:42 utils.py:140] ==================================
I 03-17 23:00:02 utils.py:131] === Checking the job status... ===
I 03-17 23:00:03 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:00:03 utils.py:140] ==================================
I 03-17 23:00:23 utils.py:131] === Checking the job status... ===
I 03-17 23:00:24 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:00:24 utils.py:140] ==================================
I 03-17 23:00:44 utils.py:131] === Checking the job status... ===
I 03-17 23:00:45 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:00:45 utils.py:140] ==================================
I 03-17 23:01:06 utils.py:131] === Checking the job status... ===
I 03-17 23:01:07 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:01:07 utils.py:140] ==================================
I 03-17 23:01:27 utils.py:131] === Checking the job status... ===
I 03-17 23:01:28 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:01:28 utils.py:140] ==================================
I 03-17 23:01:48 utils.py:131] === Checking the job status... ===
I 03-17 23:01:49 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:01:49 utils.py:140] ==================================
I 03-17 23:02:09 utils.py:131] === Checking the job status... ===
I 03-17 23:02:10 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:02:10 utils.py:140] ==================================
I 03-17 23:02:30 utils.py:131] === Checking the job status... ===
I 03-17 23:02:31 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:02:31 utils.py:140] ==================================
I 03-17 23:02:51 utils.py:131] === Checking the job status... ===
I 03-17 23:02:52 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:02:52 utils.py:140] ==================================
I 03-17 23:03:12 utils.py:131] === Checking the job status... ===
I 03-17 23:03:13 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:03:13 utils.py:140] ==================================
I 03-17 23:03:34 utils.py:131] === Checking the job status... ===
I 03-17 23:03:35 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:03:35 utils.py:140] ==================================
I 03-17 23:03:55 utils.py:131] === Checking the job status... ===
I 03-17 23:03:56 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:03:56 utils.py:140] ==================================
I 03-17 23:04:16 utils.py:131] === Checking the job status... ===
I 03-17 23:04:17 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:04:17 utils.py:140] ==================================
I 03-17 23:04:37 utils.py:131] === Checking the job status... ===
I 03-17 23:04:38 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:04:38 utils.py:140] ==================================
I 03-17 23:04:58 utils.py:131] === Checking the job status... ===
I 03-17 23:04:59 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:04:59 utils.py:140] ==================================
I 03-17 23:05:19 utils.py:131] === Checking the job status... ===
I 03-17 23:05:20 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:05:20 utils.py:140] ==================================
I 03-17 23:05:40 utils.py:131] === Checking the job status... ===
I 03-17 23:05:41 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:05:41 utils.py:140] ==================================
I 03-17 23:06:01 utils.py:131] === Checking the job status... ===
I 03-17 23:06:02 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:06:02 utils.py:140] ==================================
I 03-17 23:06:22 utils.py:131] === Checking the job status... ===
I 03-17 23:06:23 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:06:23 utils.py:140] ==================================
I 03-17 23:06:43 utils.py:131] === Checking the job status... ===
I 03-17 23:06:44 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:06:44 utils.py:140] ==================================
I 03-17 23:07:05 utils.py:131] === Checking the job status... ===
I 03-17 23:07:06 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:07:06 utils.py:140] ==================================
I 03-17 23:07:26 utils.py:131] === Checking the job status... ===
I 03-17 23:07:27 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:07:27 utils.py:140] ==================================
I 03-17 23:07:47 utils.py:131] === Checking the job status... ===
I 03-17 23:07:48 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:07:48 utils.py:140] ==================================
I 03-17 23:08:08 utils.py:131] === Checking the job status... ===
I 03-17 23:08:09 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:08:09 utils.py:140] ==================================
I 03-17 23:08:29 utils.py:131] === Checking the job status... ===
I 03-17 23:08:30 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:08:30 utils.py:140] ==================================
I 03-17 23:08:50 utils.py:131] === Checking the job status... ===
I 03-17 23:08:51 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:08:51 utils.py:140] ==================================
I 03-17 23:09:11 utils.py:131] === Checking the job status... ===
I 03-17 23:09:12 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:09:12 utils.py:140] ==================================
I 03-17 23:09:32 utils.py:131] === Checking the job status... ===
I 03-17 23:09:33 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:09:33 utils.py:140] ==================================
I 03-17 23:09:54 utils.py:131] === Checking the job status... ===
I 03-17 23:09:55 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:09:55 utils.py:140] ==================================
I 03-17 23:10:15 utils.py:131] === Checking the job status... ===
I 03-17 23:10:16 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:10:16 utils.py:140] ==================================
I 03-17 23:10:36 utils.py:131] === Checking the job status... ===
I 03-17 23:10:37 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:10:37 utils.py:140] ==================================
I 03-17 23:10:57 utils.py:131] === Checking the job status... ===
I 03-17 23:10:58 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:10:58 utils.py:140] ==================================
I 03-17 23:11:18 utils.py:131] === Checking the job status... ===
I 03-17 23:11:19 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:11:19 utils.py:140] ==================================
I 03-17 23:11:39 utils.py:131] === Checking the job status... ===
I 03-17 23:11:40 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:11:40 utils.py:140] ==================================
I 03-17 23:12:00 utils.py:131] === Checking the job status... ===
I 03-17 23:12:01 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:12:01 utils.py:140] ==================================
I 03-17 23:12:22 utils.py:131] === Checking the job status... ===
I 03-17 23:12:23 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:12:23 utils.py:140] ==================================
I 03-17 23:12:43 utils.py:131] === Checking the job status... ===
I 03-17 23:12:44 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:12:44 utils.py:140] ==================================
I 03-17 23:13:04 utils.py:131] === Checking the job status... ===
I 03-17 23:13:05 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:13:05 utils.py:140] ==================================
I 03-17 23:13:25 utils.py:131] === Checking the job status... ===
I 03-17 23:13:26 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:13:26 utils.py:140] ==================================
I 03-17 23:13:46 utils.py:131] === Checking the job status... ===
I 03-17 23:13:47 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:13:47 utils.py:140] ==================================
I 03-17 23:14:07 utils.py:131] === Checking the job status... ===
I 03-17 23:14:08 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:14:08 utils.py:140] ==================================
I 03-17 23:14:28 utils.py:131] === Checking the job status... ===
I 03-17 23:14:29 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:14:29 utils.py:140] ==================================
I 03-17 23:14:49 utils.py:131] === Checking the job status... ===
I 03-17 23:14:50 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:14:50 utils.py:140] ==================================
I 03-17 23:15:11 utils.py:131] === Checking the job status... ===
I 03-17 23:15:12 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:15:12 utils.py:140] ==================================
I 03-17 23:15:32 utils.py:131] === Checking the job status... ===
I 03-17 23:15:33 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:15:33 utils.py:140] ==================================
I 03-17 23:15:53 utils.py:131] === Checking the job status... ===
I 03-17 23:15:54 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:15:54 utils.py:140] ==================================
I 03-17 23:16:14 utils.py:131] === Checking the job status... ===
I 03-17 23:16:15 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:16:15 utils.py:140] ==================================
I 03-17 23:16:35 utils.py:131] === Checking the job status... ===
I 03-17 23:16:36 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:16:36 utils.py:140] ==================================
I 03-17 23:16:56 utils.py:131] === Checking the job status... ===
I 03-17 23:16:57 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:16:57 utils.py:140] ==================================
I 03-17 23:17:17 utils.py:131] === Checking the job status... ===
I 03-17 23:17:18 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:17:18 utils.py:140] ==================================
I 03-17 23:17:38 utils.py:131] === Checking the job status... ===
I 03-17 23:17:39 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:17:39 utils.py:140] ==================================
I 03-17 23:17:59 utils.py:131] === Checking the job status... ===
I 03-17 23:18:00 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:18:00 utils.py:140] ==================================
I 03-17 23:18:21 utils.py:131] === Checking the job status... ===
I 03-17 23:18:22 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:18:22 utils.py:140] ==================================
I 03-17 23:18:42 utils.py:131] === Checking the job status... ===
I 03-17 23:18:43 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:18:43 utils.py:140] ==================================
I 03-17 23:19:03 utils.py:131] === Checking the job status... ===
I 03-17 23:19:04 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:19:04 utils.py:140] ==================================
I 03-17 23:19:24 utils.py:131] === Checking the job status... ===
I 03-17 23:19:25 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:19:25 utils.py:140] ==================================
I 03-17 23:19:45 utils.py:131] === Checking the job status... ===
I 03-17 23:19:46 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:19:46 utils.py:140] ==================================
I 03-17 23:20:06 utils.py:131] === Checking the job status... ===
I 03-17 23:20:07 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:20:07 utils.py:140] ==================================
I 03-17 23:20:27 utils.py:131] === Checking the job status... ===
I 03-17 23:20:28 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:20:28 utils.py:140] ==================================
I 03-17 23:20:49 utils.py:131] === Checking the job status... ===
I 03-17 23:20:50 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:20:50 utils.py:140] ==================================
I 03-17 23:21:10 utils.py:131] === Checking the job status... ===
I 03-17 23:21:11 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:21:11 utils.py:140] ==================================
I 03-17 23:21:31 utils.py:131] === Checking the job status... ===
I 03-17 23:21:32 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:21:32 utils.py:140] ==================================
I 03-17 23:21:52 utils.py:131] === Checking the job status... ===
I 03-17 23:21:53 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:21:53 utils.py:140] ==================================
I 03-17 23:22:13 utils.py:131] === Checking the job status... ===
I 03-17 23:22:14 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:22:14 utils.py:140] ==================================
I 03-17 23:22:34 utils.py:131] === Checking the job status... ===
I 03-17 23:22:35 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:22:35 utils.py:140] ==================================
I 03-17 23:22:55 utils.py:131] === Checking the job status... ===
I 03-17 23:22:56 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:22:56 utils.py:140] ==================================
I 03-17 23:23:16 utils.py:131] === Checking the job status... ===
I 03-17 23:23:17 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:23:17 utils.py:140] ==================================
I 03-17 23:23:37 utils.py:131] === Checking the job status... ===
I 03-17 23:23:38 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:23:38 utils.py:140] ==================================
I 03-17 23:23:59 utils.py:131] === Checking the job status... ===
I 03-17 23:24:00 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:24:00 utils.py:140] ==================================
I 03-17 23:24:20 utils.py:131] === Checking the job status... ===
I 03-17 23:24:21 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:24:21 utils.py:140] ==================================
I 03-17 23:24:41 utils.py:131] === Checking the job status... ===
I 03-17 23:24:42 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:24:42 utils.py:140] ==================================
I 03-17 23:25:02 utils.py:131] === Checking the job status... ===
I 03-17 23:25:03 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:25:03 utils.py:140] ==================================
I 03-17 23:25:23 utils.py:131] === Checking the job status... ===
I 03-17 23:25:24 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:25:24 utils.py:140] ==================================
I 03-17 23:25:44 utils.py:131] === Checking the job status... ===
I 03-17 23:25:45 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:25:45 utils.py:140] ==================================
I 03-17 23:26:05 utils.py:131] === Checking the job status... ===
I 03-17 23:26:06 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:26:06 utils.py:140] ==================================
I 03-17 23:26:26 utils.py:131] === Checking the job status... ===
I 03-17 23:26:27 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:26:27 utils.py:140] ==================================
I 03-17 23:26:47 utils.py:131] === Checking the job status... ===
I 03-17 23:26:48 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:26:48 utils.py:140] ==================================
I 03-17 23:27:09 utils.py:131] === Checking the job status... ===
I 03-17 23:27:10 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:27:10 utils.py:140] ==================================
I 03-17 23:27:30 utils.py:131] === Checking the job status... ===
I 03-17 23:27:31 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:27:31 utils.py:140] ==================================
I 03-17 23:27:51 utils.py:131] === Checking the job status... ===
I 03-17 23:27:52 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:27:52 utils.py:140] ==================================
I 03-17 23:28:12 utils.py:131] === Checking the job status... ===
I 03-17 23:28:13 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:28:13 utils.py:140] ==================================
I 03-17 23:28:33 utils.py:131] === Checking the job status... ===
I 03-17 23:28:34 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:28:34 utils.py:140] ==================================
I 03-17 23:28:54 utils.py:131] === Checking the job status... ===
I 03-17 23:28:55 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:28:55 utils.py:140] ==================================
I 03-17 23:29:15 utils.py:131] === Checking the job status... ===
I 03-17 23:29:16 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:29:16 utils.py:140] ==================================
I 03-17 23:29:36 utils.py:131] === Checking the job status... ===
I 03-17 23:29:37 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:29:37 utils.py:140] ==================================
I 03-17 23:29:57 utils.py:131] === Checking the job status... ===
I 03-17 23:29:58 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:29:58 utils.py:140] ==================================
I 03-17 23:30:18 utils.py:131] === Checking the job status... ===
I 03-17 23:30:19 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:30:19 utils.py:140] ==================================
I 03-17 23:30:40 utils.py:131] === Checking the job status... ===
I 03-17 23:30:41 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:30:41 utils.py:140] ==================================
I 03-17 23:31:01 utils.py:131] === Checking the job status... ===
I 03-17 23:31:02 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:31:02 utils.py:140] ==================================
I 03-17 23:31:22 utils.py:131] === Checking the job status... ===
I 03-17 23:31:23 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:31:23 utils.py:140] ==================================
I 03-17 23:31:43 utils.py:131] === Checking the job status... ===
I 03-17 23:31:44 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:31:44 utils.py:140] ==================================
I 03-17 23:32:04 utils.py:131] === Checking the job status... ===
I 03-17 23:32:05 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:32:05 utils.py:140] ==================================
I 03-17 23:32:25 utils.py:131] === Checking the job status... ===
I 03-17 23:32:26 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:32:26 utils.py:140] ==================================
I 03-17 23:32:46 utils.py:131] === Checking the job status... ===
I 03-17 23:32:47 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:32:47 utils.py:140] ==================================
I 03-17 23:33:07 utils.py:131] === Checking the job status... ===
I 03-17 23:33:08 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:33:08 utils.py:140] ==================================
I 03-17 23:33:29 utils.py:131] === Checking the job status... ===
I 03-17 23:33:30 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:33:30 utils.py:140] ==================================
I 03-17 23:33:50 utils.py:131] === Checking the job status... ===
I 03-17 23:33:51 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:33:51 utils.py:140] ==================================
I 03-17 23:34:11 utils.py:131] === Checking the job status... ===
I 03-17 23:34:12 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:34:12 utils.py:140] ==================================
I 03-17 23:34:32 utils.py:131] === Checking the job status... ===
I 03-17 23:34:33 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:34:33 utils.py:140] ==================================
I 03-17 23:34:53 utils.py:131] === Checking the job status... ===
I 03-17 23:34:54 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:34:54 utils.py:140] ==================================
I 03-17 23:35:14 utils.py:131] === Checking the job status... ===
I 03-17 23:35:15 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:35:15 utils.py:140] ==================================
I 03-17 23:35:35 utils.py:131] === Checking the job status... ===
I 03-17 23:35:36 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:35:36 utils.py:140] ==================================
I 03-17 23:35:56 utils.py:131] === Checking the job status... ===
I 03-17 23:35:57 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:35:57 utils.py:140] ==================================
I 03-17 23:36:18 utils.py:131] === Checking the job status... ===
I 03-17 23:36:19 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:36:19 utils.py:140] ==================================
I 03-17 23:36:39 utils.py:131] === Checking the job status... ===
I 03-17 23:36:40 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:36:40 utils.py:140] ==================================
I 03-17 23:37:00 utils.py:131] === Checking the job status... ===
I 03-17 23:37:01 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:37:01 utils.py:140] ==================================
I 03-17 23:37:21 utils.py:131] === Checking the job status... ===
I 03-17 23:37:22 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:37:22 utils.py:140] ==================================
I 03-17 23:37:42 utils.py:131] === Checking the job status... ===
I 03-17 23:37:43 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:37:43 utils.py:140] ==================================
I 03-17 23:38:03 utils.py:131] === Checking the job status... ===
I 03-17 23:38:04 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:38:04 utils.py:140] ==================================
I 03-17 23:38:24 utils.py:131] === Checking the job status... ===
I 03-17 23:38:25 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:38:25 utils.py:140] ==================================
I 03-17 23:38:45 utils.py:131] === Checking the job status... ===
I 03-17 23:38:46 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:38:46 utils.py:140] ==================================
I 03-17 23:39:07 utils.py:131] === Checking the job status... ===
I 03-17 23:39:08 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:39:08 utils.py:140] ==================================
I 03-17 23:39:28 utils.py:131] === Checking the job status... ===
I 03-17 23:39:29 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:39:29 utils.py:140] ==================================
I 03-17 23:39:49 utils.py:131] === Checking the job status... ===
I 03-17 23:39:50 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:39:50 utils.py:140] ==================================
I 03-17 23:40:10 utils.py:131] === Checking the job status... ===
I 03-17 23:40:11 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:40:11 utils.py:140] ==================================
I 03-17 23:40:31 utils.py:131] === Checking the job status... ===
I 03-17 23:40:32 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:40:32 utils.py:140] ==================================
I 03-17 23:40:52 utils.py:131] === Checking the job status... ===
I 03-17 23:40:53 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:40:53 utils.py:140] ==================================
I 03-17 23:41:13 utils.py:131] === Checking the job status... ===
I 03-17 23:41:14 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:41:14 utils.py:140] ==================================
I 03-17 23:41:35 utils.py:131] === Checking the job status... ===
I 03-17 23:41:36 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:41:36 utils.py:140] ==================================
I 03-17 23:41:56 utils.py:131] === Checking the job status... ===
I 03-17 23:41:57 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:41:57 utils.py:140] ==================================
I 03-17 23:42:17 utils.py:131] === Checking the job status... ===
I 03-17 23:42:18 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:42:18 utils.py:140] ==================================
I 03-17 23:42:38 utils.py:131] === Checking the job status... ===
I 03-17 23:42:39 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:42:39 utils.py:140] ==================================
I 03-17 23:42:59 utils.py:131] === Checking the job status... ===
I 03-17 23:43:00 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:43:00 utils.py:140] ==================================
I 03-17 23:43:20 utils.py:131] === Checking the job status... ===
I 03-17 23:43:21 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:43:21 utils.py:140] ==================================
I 03-17 23:43:41 utils.py:131] === Checking the job status... ===
I 03-17 23:43:42 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:43:42 utils.py:140] ==================================
I 03-17 23:44:02 utils.py:131] === Checking the job status... ===
I 03-17 23:44:03 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:44:03 utils.py:140] ==================================
I 03-17 23:44:23 utils.py:131] === Checking the job status... ===
I 03-17 23:44:24 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:44:24 utils.py:140] ==================================
I 03-17 23:44:45 utils.py:131] === Checking the job status... ===
I 03-17 23:44:46 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:44:46 utils.py:140] ==================================
I 03-17 23:45:06 utils.py:131] === Checking the job status... ===
I 03-17 23:45:07 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:45:07 utils.py:140] ==================================
I 03-17 23:45:27 utils.py:131] === Checking the job status... ===
I 03-17 23:45:28 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:45:28 utils.py:140] ==================================
I 03-17 23:45:48 utils.py:131] === Checking the job status... ===
I 03-17 23:45:49 utils.py:137] Job status: JobStatus.RUNNING
I 03-17 23:45:49 utils.py:140] ==================================
